### DISCUSSION TITLE
How to disable favicons

### DISCUSSION THREAD

**Discussion by sorairolake** [jpanther/congo#711] [Q&A]
I provided an empty `layouts/partials/favicons.html` for disable favicons based on the [documentation](https://jpanther.github.io/congo/docs/partials/#favicons). I thought this would work, but the default favicons were displayed.

After that, I removed a `layouts/partials/favicons.html` and placed empty icon assets in the `static/` and it seemed to work. Is this the right way to disable favicons? Or is there another way?

**Comment by wolfspyre** [comment]
I THINK the problem relates to the fact that the assets are stored in the theme's `static` directory, which is published by default.

I suspect the ultimate fix is to relocate the theme's favicon assets to assets/favicon or something, and publish them rather than just having them automagically added to the site's content.

**Comment by jpanther** [comment]
This is happening because most browsers will pick up the favicon file regardless of whether you specify it in your HTML. It's basically how favicons are meant to work and so the way to disable it is by simply not including them in the site output... something that is not easy with the way this is currently implemented so providing an empty file in the static folder is currently the best workaround.

**Reply by jpanther** [reply]
I think functionality to disable the favicon could be a useful feature so I've included this in #720.
